apis/workloads/v1alpha1/lifecycle_recorder.go

```diff
@@ -45,6 +47,28 @@ type consoleIdBuilderImpl struct {
 	contextName string
 }
 
+var (
+	lifecycleEventsPublish = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "theatre_lifecycle_events_published",
```

Comment:
Don't forget to add `_total` to counters:

https://prometheus.io/docs/practices/naming/